### ISSUE TITLE
Unable to navigate screen

### ISSUE THREAD

**Issue by thennant** [thennant/pe#2]
Upon opening the app, if I were to press "h", the screen blanks out and I am unable to navigate using the "l" command. Similarly, if i were to press "l" first, the screen also blanks out and I am unable to navigate back using the "h" command. I can only do the ":q!" command to quit the app and open the app again.

First time opening the app:
![image.png](https://raw.githubusercontent.com/thennant/pe/main/files/7aea721d-63e8-4502-a618-7ebc863bfca7.png)

After pressing "h":
![image.png](https://raw.githubusercontent.com/thennant/pe/main/files/49279727-e9c2-4138-909d-2821a32b73e8.png)
Pressing "l" or "h" here does not do anything

Tldr, having a home command that can bring the user back to the main screen would be more convenient for the user

**Comment by nus-pe-script** [comment]
# Team's Response
The welcome panel is not an instruction manual. Instead, users should refer to the user guide on how to use the application. Upon further reading, the user will know how to add a task, and proceed with normal usage.
Furthermore, we do not understand what is the main screen you are referring to.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]

- [x] I disagree

**Reason for disagreement:** The main screen in this case is what the user sees when opening the app (the first image). When I first open the app, the first thing i saw was the navigating around commands "h", "j", "k", "l". Upon clicking either "h" or  "l" command, the screen just disappears (the second image). The only way i could get back was by quitting the app and opening it again. There was no specific instruction to inform the user what to do if the main screen were to turn blank in the UG  (the second image in the issue) so imagine if I was a first-time user or someone who has little to no experience in vim and trying the app for the first time, I would be lost and unsure what to do or how to get back to the main screen upon seeing the blank screen.

<catcher-end-of-segment><hr>
